gapic/schema/api.py

```diff
@@ -804,6 +844,7 @@ def _load_message(self,
                 address=address,
                 documentation=self.docs.get(path, self.EMPTY),
             ),
+            oneofs=oneofs,
```

Comment:
@software-dov  I didn't use this anywhere, but oneofs are part of a message so I thought having it here may serve useful

Reply:
Can't hurt.